merge_phase_jsons.py

```
@@ -0,0 +1,289 @@
+    if not pairs:
+        print("No matching Phase 1 and Phase 2 pairs found")
+        sys.exit(1)
+    
+    print(f"\nFound {len(pairs)} pairs to merge")
+    
+    # 各ペアを処理
+    successful_merges = 0
+    for phase1_path, phase2_path in pairs:
+        try:
+            print(f"\nMerging: {phase1_path.name} + {phase2_path.name}")
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)